test/LocalizationWebsite/StartupResourcesAtRootFolder.cs

```diff
@@ -26,7 +29,17 @@ public void ConfigureServices(IServiceCollection services)
         {
             loggerFactory.AddConsole(minLevel: LogLevel.Warning);
 
-            app.UseRequestLocalization();
+            var options = new RequestLocalizationOptions();
+            options.SupportedCultures = new List<CultureInfo>()
+            {
+                new CultureInfo("fr-FR")
+            };
+            options.SupportedUICultures = new List<CultureInfo>()
+            {
+                new CultureInfo("fr-FR")
+            };
+
+            app.UseRequestLocalization(options, new RequestCulture(new CultureInfo("en-US")));
```

Comment:
Also kind of seems like maybe `RequestCulture` should have a ctor that takes in a culture name - wouldn't that save a bit of verbose code?